### ISSUE TITLE
Solarus Quest Editor 1.6 UI: Multiple tilesets feature

### ISSUE THREAD

**Issue by maxamine** [christopho/solarus-quest-editor#361]
I thought about a better presentation on the UI for the "multiple tilesets" feature.
see the screenshots below:

![1](https://user-images.githubusercontent.com/14140919/39676384-db15abee-5161-11e8-8cf8-24560ae09e1d.png)

![2](https://user-images.githubusercontent.com/14140919/39676387-f139c1f8-5161-11e8-8f95-ffaea3e3a2c4.png)

![3](https://user-images.githubusercontent.com/14140919/39676396-113dd8a4-5162-11e8-8854-45cbf58faeda.png)

![4](https://user-images.githubusercontent.com/14140919/39676404-2adcc054-5162-11e8-8b18-05318b0ce1d0.png)

**Comment by maxamine** [comment]
A delete button can be added next to each tab to allow removing tilesets affiliated to the map. (except for the default map)

**Comment by Diarandor** [comment]
If you remove a tileset, you should allow to remove all the tiles of that tileset too.

**Comment by maxamine** [comment]
Diarandor: we can also replace these tiles with missing tiles png

**Comment by Diarandor** [comment]
Yes, but what I meant is that when you remove a tileset, all the associated tiles should be removed automatically too.

**Comment by PhoenixII54** [comment]
-or- ask user what to do with these tiles (remove/keep as unassigned)
